engine/runtime/src/main/scala/org/enso/compiler/Cache.scala

```diff
@@ -0,0 +1,463 @@
+package org.enso.compiler
+
+import com.oracle.truffle.api.{TruffleFile, TruffleLogger}
+import org.bouncycastle.jcajce.provider.digest.SHA3
```

Comment:
Curious: Why do we use bouncycastle and not JDK's own `MessageDigest` implementations?

Reply:
No idea. I just stick to whatever is there and works.